### PR TITLE
initial config for ddb tables

### PR DESCRIPTION
**Pull request by itsthatriver**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -88,22 +88,40 @@ functions:
...
+        TableName: 52calls-52weeks-representatives
+        KeySchema:
+          - AttributeName: district
+            KeyType: HASH
+        AttributeDefinitions:
+          - AttributeName: district
+            AttributeType: S
+        ProvisionedThroughput:
+          ReadCapacityUnits: 5
+          WriteCapacityUnits: 5
```
Comment (jontg) [inline serverless.yml:123]: @sethvoltz does it make sense to turn down `WriteCapacityUnits` if we know that the write-through on this table is low?

File: serverless.yml [hunk 2]
```diff
@@ -88,22 +88,40 @@ functions:
...
-#         ProvisionedThroughput:
-#           ReadCapacityUnits: 5
-#           WriteCapacityUnits: 5
-#
+resources:
+  Resources:
+    usersTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: 52calls-52weeks-users
```
Comment (jontg) [inline serverless.yml]: I *believe* we can change the name here to just `users` without loss of generality; lets give it a try.

I'm assuming serverless already does this, but we'll probably want to incorporate `${opt:stage}` into the name if it doesn't.  Once we get this to staging we can fast-follow to figure out if the name is scoped down somehow.

**Review by jontg — APPROVED** [review]

File: serverless.yml (hunk 2 above)
Comment (itsthatriver) [reply serverless.yml]: works fine for me, I can make that change once I get back to it

File: serverless.yml (hunk 1 above)
Comment (itsthatriver) [reply serverless.yml:123]: The impression I got from chatting with Seth yesterday was that you can overprovision bandwidth without getting linear correlation in your costs as long as you're not saturating it. Apparently there's a cost estimator that can help with this, but if that's the case I'd lean toward going higher on this and cutting the time of our batch uploads and reads in half. Although, grand scale, as long as we have queueing, shouldn't really matter, and we can easily reconfigure later with a redeploy

Comment (itsthatriver) [reply serverless.yml:123]: the example seth gave was ~97 cents a month for 5/5, or ~98 cents a month for 10/10 given a low saturation, bursty workload.

**Pull request merged**